pages/resources/jobExplorer/devOpsEngineerProject.tsx

```diff
@@ -1,20 +0,0 @@
-import React from "react";
-import LandingNavbar from "../../../components/landingPage/LandingNavbar";
-import DevOpsProject from "../../../components/resources/jobExplorer/devOpsEngineer/DevOpsProject";
-
-export default function DOProjectPage() {
```

Comment:
we don't need this page anymore, because we're going to use a generic page that uses slugs to fetch the right project data